packages/server/lib/util/settings.js

```diff
@@ -3,7 +3,36 @@ const Promise = require('bluebird')
 const path = require('path')
 const errors = require('../errors')
 const log = require('../log')
-const { fs } = require('../util/fs')
+const { fs } = require('./fs')
+const requireAsync = require('./require_async').default
+const { CYPRESS_CONFIG_FILES } = require('../configFiles')
+const debug = require('debug')('cypress:server:settings')
+
+function jsCode (obj) {
+  const objJSON = obj && !_.isEmpty(obj)
+    ? JSON.stringify(_.omit(obj, 'configFile'), null, 2)
+    : `{
+
+}`
+
+  return `const { defineConfig } = require('cypress')
+
+module.export = defineConfig(${objJSON})
+`
+}
+
+function tsCode (obj) {
```

Comment:
If we export the `ConfigOptions` type, can we also update any example `cypress.config.ts` file we generate to feel a little more like standard Typescript, eg.

```ts
import type { ConfigOptions } from 'cypress'

const config: ConfigOptions = {
  
}

export default config
```